graphql-dgs-spring-webmvc/src/main/kotlin/com/netflix/graphql/dgs/mvc/DgsRestController.kt

```diff
@@ -91,23 +92,32 @@ open class DgsRestController(
         @RequestParam(name = "map") mapParam: String?,
         @RequestHeader headers: HttpHeaders,
         webRequest: WebRequest
-    ): ResponseEntity<String> {
+    ): ResponseEntity<StreamingResponseBody> {
+
```

Comment:
I think, if we want to support this, is a feature that folks need to opt-in and not have by default. I.e. We could have a `DgsRestController` and a `DgsStreamingRestController`. People could opt in and then, as part of the process, define the `WebMvcConfigurer`'s `ThreadPoolTaskExecutor`.